Userland/Libraries/LibWeb/HTML/HTMLVideoElement.cpp

```diff
@@ -24,4 +24,20 @@ JS::ThrowCompletionOr<void> HTMLVideoElement::initialize(JS::Realm& realm)
     return {};
 }
 
+// https://html.spec.whatwg.org/multipage/media.html#dom-video-videowidth
+u32 HTMLVideoElement::video_width() const
+{
+    if (ready_state() == ReadyState::HaveNothing)
```

Comment:
`// The videoWidth IDL attribute must return the intrinsic width of the video in CSS pixels. The videoHeight IDL attribute must return the intrinsic height of the video in CSS pixels. If the element's readyState attribute is HAVE_NOTHING, then the attributes must return 0.`